### PR TITLE
NRPT-788 and NRPT-809

### PR DESCRIPTION
**Pull request by cameronpettit**
NRPT-809 populate description field in collapsed row

NRPT-788 changes to card layouts, warnings added

NRPT-809 moving outcome description field

NRPT-788 and NRPT-809

https://bcmines.atlassian.net/browse/NRPT-788
https://bcmines.atlassian.net/browse/NRPT-809

### CODE REVIEW

File: angular/projects/public-nrpti/src/app/records/restorative-justices/restorative-justice-detail/restorative-justice-detail.component.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
       <div class="content-grid">
         <div class="grid-section section-1">
           <div class="grid-item__row">
-            <label class="grid-item-name">Issued To</label>
+            <label class="grid-item-name">Date Issued</label>
```
Comment (BcGovNeal) [inline angular/projects/public-nrpti/src/app/records/restorative-justices/restorative-justice-detail/restorative-justice-detail.component.html:20]: Should be Issued To here

File: angular/projects/public-nrpti/src/app/records/tickets/ticket-detail/ticket-detail.component.html [hunk 2]
```diff
@@ -17,11 +17,7 @@
       <div class="content-grid">
         <div class="grid-section section-1">
           <div class="grid-item__row">
-            <label class="grid-item-name">Type</label>
-            <span class="grid-item-value__col">{{ (data && data.recordType) || '-' }}</span>
-          </div>
-          <div class="grid-item__row">
-            <label class="grid-item-name">Issued To</label>
+            <label class="grid-item-name">Date Issued</label>
```
Comment (BcGovNeal) [inline angular/projects/public-nrpti/src/app/records/tickets/ticket-detail/ticket-detail.component.html:20]: Should be Issued To

File: angular/projects/public-nrpti/src/app/records/warnings/warning-detail/warning-detail.component.html [hunk 3]
```diff
@@ -17,11 +17,7 @@
       <div class="content-grid">
         <div class="grid-section section-1">
           <div class="grid-item__row">
-            <label class="grid-item-name">Type</label>
-            <span class="grid-item-value__col">{{ (data && data.recordType) || '-' }}</span>
-          </div>
-          <div class="grid-item__row">
-            <label class="grid-item-name">Issued To</label>
+            <label class="grid-item-name">Date Issued</label>
```
Comment (BcGovNeal) [inline angular/projects/public-nrpti/src/app/records/warnings/warning-detail/warning-detail.component.html:20]: Should be Issued To

**Review by BcGovNeal — CHANGES_REQUESTED** [review]
Got some mislabeled fields

**Review by BcGovNeal — APPROVED** [review]

**Review by marklise — APPROVED** [review]

**Pull request merged**